pahfit/base.py

```diff
@@ -0,0 +1,462 @@
+from astropy.modeling.functional_models import Gaussian1D
+
+from pahfit.component_models import (BlackBody1D, Drude1D,
+                                     S07_attenuation)
+
+from astropy.table import Table, vstack
+
+import numpy as np
+
+__all__ = ['PAHFITBase']
+
+
+def _ingest_limits(min_vals, max_vals):
+    """
+    Ingest the limits read from a file and generate the appropriate
+    internal format (list of tuples).  Needed to handle the case
+    where a limit is not desired as numpy arrays cannot have elements
+    of None, instead a value of nan is used.
+
+    Limits that are not set are designated as 'nan' in files and
+    these are changed to the python None to be compatible with
+    the astropy.modeling conventation.
```

Comment:
Shouldn't "the astropy.modeling conventation" be "the astropy.modeling convention"?

Reply:
Yes.  Fixed.